server/routes/statistics.js

```diff
@@ -66,6 +66,12 @@ module.exports = function (config) {
 	getData(driver).done();
 
 	return assign({
+		'get-flow-data': function (query) {
+			return queryHandler.resolve(query)(function (query) {
```

Comment:
We should use different queryHandler for this request, query options for requests are different, and validation rules are different

In `queryHandler` as provided here, we miss proper normalization and validation of e.g. `page`. so it's count does not overflow the pages count (e.g. see one made in business process table query handler: https://github.com/egovernment/eregistrations/blob/master/view/components/business-processes-table/setup-query-handler.js#L61-L62)

Then we should use same `queryHandler` on both _client_ and _service_ (but again: trying to reuse one configured for quite different filter bar in different view, won't work well).